Frameworks/UIKit/UIApplicationMain.mm

```diff
@@ -352,12 +355,19 @@ void _UIApplicationShutdown() {
 }
 
 extern "C" void UIApplicationMainHandleVoiceCommandEvent(IInspectable* voiceCommandResult) {
-    WMSSpeechRecognitionResult* result = [WMSSpeechRecognitionResult createWith:voiceCommandResult];
-    [[UIApplication sharedApplication] _sendVoiceCommandReceivedEvent:result];
+    id result = _createProjectionObject("WMSSpeechRecognitionResult", voiceCommandResult, "voice commands");
+
+    if (result) {
```

Comment:
perhaps we should fire the event even if 'result' is nil, so it's more clear to the app dev that they need to pull in projections?